metricflow/sql/sql_exprs.py

```diff
@@ -849,18 +851,22 @@ def from_aggregation_parameters(agg_params: MeasureAggregationParameters) -> Sql
         """Given the measure parameters, returns a SqlPercentileExpressionArgument with the corresponding percentile args."""
         if not agg_params.percentile:
             raise RuntimeError("Percentile value is none - this should have been caught during model parsing.")
+        flags_to_function_type = {
```

Comment:
This one is not easy to understand without seeing how it is used later. Can we use some type annotation to make it easier to understand?

Reply:
I was thing something like this:

```python
UseDiscretePercentile = bool
UseApproximatePercentile = bool
flags_to_function_type: Map[Tuple[UseDiscretePercentile, UseApproximatePercentile], bool]
```